data/hai/hai reveal 4 middle.txt

```diff
@@ -97,37 +96,37 @@ mission "Hai Reveal [A02] Symbol Identification"
+			`	"First, you must understand that violent crimes in Hai space are not a rare thing, though they aren't a day-to-day part of life as in human space. Murder, assault, estranged parents kidnapping their children, all of that happens here. Some Hai go on an adventure one day and then get fall in a crevasse or get eaten by a beast. Some join the Unfettered, but don't tell anyone. Most cases are solved: the person is found dead or alive. Sometimes, though, the no person or body is ever found."`
+			`	"In recent years, a disproportionate number of Hai and human kidnapping victims seem to be purposefully picked from the intelligentsia: doctors, scientists, engineers, architects. Most were specialists within their field. None were prominent enough for their departure to send shock waves in the community though. Some disappeared from worlds where we'd expect to be able to track them to their general area, but instead, they are simply... gone.`
+			`	"We believe someone is targeting researchers and specialists and trying to hide it beneath the noise."`
 				goto questions
 			label kidnappings
-			`	Teeneep answers, "In human space, we don't know, but after collating records in Hai space it seems to have happened in three spates each lasting several years. The first of which date back to only about a decade after we made contact with humans, though it doesn't seem to have been consistent in either time or method, merely in the type of people targeted. This of course means that we can't even say for sure if it's even been the same people, but every investigation has ultimately dead-ended with similar difficulties."`
-			`	"Nevertheless, from the rate at which the disappearances have happened, possibly as many as five thousand Hai have vanished this way. The number may be less; some of those cases may be undiscovered murders, secret changes of identity, or defections to the Unfettered. We have no idea how many humans, we don't have that level of access to your records and they're not that good anyway."`
+			`	Teeneep answers, "In human space, we don't know, but after collating records in Hai space it seems to have happened in three spates each lasting several years. The earliest date back to only about a decade after we made contact with humans, around 90 years, but often with different kidnapping methods and targets. This means that we can't even say for sure if it's even been the same people, but every investigation has ultimately dead-ended with similar difficulties."`
```

Comment:
```suggestion
			`	Teeneep answers, "In human space, we don't know, but after collating records in Hai space it seems to have happened in three spates each lasting several years. The earliest date back to only about a decade after we made contact with humans, around 90 years ago, but often with different kidnapping methods and targets. This means that we can't even say for sure if it's even been the same people, but every investigation has ultimately dead-ended with similar difficulties."`
```

I'm not 100% sure of this date being good, since it's not otherwise mentioned here.

Reply:
At the beginning of the wanderers campaign, the hai tell us their first contact with humans was about a century ago.

I'll remove the "around 90 years ago" to avoid this problem.

Reply:
I think that's fine as is then, just wasn't sure myself :)

Reply:
I don't want to change any lore in this PR, so 90 years has to go. This is just a dialog rearrangement PR.

I just removed it.